### ISSUE TITLE
Test failure with latest nestedtext

### ISSUE THREAD

**Issue by fabaff** [kalekundert/parametrize_from_file#19]
With the latest `nestedtext` release, there seems to be a test failure.

```bash
============================= test session starts ==============================
platform linux -- Python 3.10.9, pytest-7.2.0, pluggy-1.0.0
rootdir: /build/parametrize_from_file-0.17.0, configfile: pyproject.toml
collected 300 items                                                            

parametrize_from_file/__init__.py .....                                  [  1%]
tests/test_namespace.py ................................................ [ 17%]
........................................................................ [ 41%]
................................                                         [ 52%]
tests/test_parameters.py ..........................F.................... [ 68%]
.....................................                                    [ 80%]
tests/test_schema.py ................................................... [ 97%]
........                                                                 [100%]

=================================== FAILURES ===================================
___________ test_load_suite_params_err[load-err.nt-files5-messages5] ___________

loader = <function load at 0x7ffff66c49d0>, path = 'err.nt'
files = {'err.nt': 'a ='}
messages = ['failed to load parametrization file', 'attempted to load file with: nestedtext.load\\(\\)']
tmp_path = PosixPath('/build/pytest-of-nixbld/pytest-0/test_load_suite_params_err_loa2')

    @pytest.mark.parametrize(
            'loader, path, files, messages', [(
                lambda p: None,
                'does-not-exist.json',
                {},
                [
                    "can't find parametrization file",
                    "does-not-exist.json",
                ],
            ), (
                # Check examples of real files that the built-in loaders will fail
                # to parse, to make sure that the error messages are still good.
                pffp.get_loaders()['.json'],
                'err.json',
                {'err.json': '{"a":'},
                [
                    "failed to load parametrization file",
                    r"attempted to load file with: json.load\(\)",
                    "Expecting value",
                ],
            ), (
                pffp.get_loaders()['.yml'],
                'err.yml',
                {'err.yml': ':'},
                [
                    "failed to load parametrization file",
                    r"attempted to load file with: yaml.safe_load\(\)",
                    "expected <block end>, but found ':'",
                ],
            ), (
                pffp.get_loaders()['.yaml'],
                'err.yaml',
                {'err.yaml': ':'},
                [
                    "failed to load parametrization file",
                    r"attempted to load file with: yaml.safe_load\(\)",
                    "expected <block end>, but found ':'",
                ],
            ), (
                pffp.get_loaders()['.toml'],
                'err.toml',
                {'err.toml': 'a ='},
                [
                    "failed to load parametrization file",
                    r"attempted to load file with: toml.decoder.load\(\)",
                    "Empty value is invalid",
                ],
            ), (
                pffp.get_loaders()['.nt'],
                'err.nt',
                {'err.nt': 'a ='},
                [
                    "failed to load parametrization file",
                    r"attempted to load file with: nestedtext.load\(\)",
                ],
            ), (
                # Any braces in error messages must not be evaluated by tidyexc.
                value_error_with_braces,
                'braces.xyz',
                {'braces.xyz': ''},
                [
                    "failed to load parametrization file",
                    "attempted to load file with: .*value_error_with_braces",
                    r"\{hello world\}",
                ],
            ),
        ]
    )
    def test_load_suite_params_err(loader, path, files, messages, tmp_path):
        for p, contents in files.items():
            (tmp_path / p).write_text(contents)
    
        with pytest.raises(pff.ConfigError) as err:
            pffp._load_and_cache_suite_params(loader, tmp_path / path)
    
        for msg in messages:
>           assert err.match(msg)
E           AssertionError: Regex pattern did not match.
E            Regex: 'attempted to load file with: nestedtext.load\\(\\)'
E            Input: 'failed to load parametrization file\n• attempted to load file with: nestedtext.nestedtext.load()\n✖ /build/pytest-of-nixbld/pytest-0/test_load_suite_params_err_loa2/err.nt, 1: unrecognized line.\n'

tests/test_parameters.py:269: AssertionError
=========================== short test summary info ============================
```

I'm not familiar with the code base of `parametrize_from_file` thus all help would be appreciated. Thanks.

**Comment by kalekundert** [comment]
Sorry for the slow reply; for some reason I didn't notice this issue when you originally posted it.  This test failure is just due to a change in the way a certain error message is worded, so it's not something that will adversely affect users.  It was fixed by 1fb2b7b, but I haven't made a new release with that commit yet since there haven't been any user-facing changes since then.  If this is causing problems for you, though, let me know and I can make a new release.

**Comment by kalekundert** [comment]
I'm closing this issue now that I've finally made a new release.  Let me know if you are still having problems.
